src/TodoListItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function TodoListItem(props) {
+
+    return (
+    <div>
```

Comment:
Just to keep the DOM lean and mean, could we try removing the div that's wrapping all the content? I could be mistaken, but I think some screen readers are easier to understand when a li tag is directly underneath a ul tag. If there's any style issues that come from this, we can target the li with a class and add some styling!